packages/storage-plus/src/de.rs

```diff
@@ -83,6 +119,11 @@ impl KeyDeserialize for Addr {
     fn from_slice(value: &[u8]) -> StdResult<Self::Output> {
         Ok(Addr::unchecked(String::from_slice(value)?))
     }
+
+    #[inline(always)]
+    fn from_vec(value: Vec<u8>) -> StdResult<Self::Output> {
+        Addr::from_slice(value.as_slice())
```

Comment:
`String::from_utf8(value)` ?

Reply:
`Addr::from_vec`, just to keep the base impl unique.